scanamo/src/main/scala/org/scanamo/DynamoObject.scala

```diff
@@ -21,11 +24,14 @@ sealed abstract class DynamoObject extends Product with Serializable { self =>
     */
   final def apply(key: String): Option[DynamoValue] = self match {
     case Empty          => None
-    case Strict(xs)     => Option(xs.get(key)).map(DynamoValue.fromAttributeValue)
+    case Strict(xs)     => val x = xs.get(key); if (x eq null) None else Some(DynamoValue.fromAttributeValue(x))
```

Comment:
Isn't `Option().map` the same?

Reply:
Semantically, yes. Operationally, there is less allocation (the initial `Option(xs.get(key))` is created on the heap and thrown away immediately).